frontend/src/components/camp/CampCategories.vue

```diff
@@ -70,14 +70,21 @@ Displays all periods of a single camp and allows to edit them & create new ones
                     <button-delete v-on="on" />
                   </template>
                   <template v-if="findActivities(category).length > 0" #error>
-                    {{ $tc('components.camp.CampCategories.deleteCategoryNotPossibleInUse') }}
+                    {{ $tc('components.camp.campCategories.deleteCategoryNotPossibleInUse') }}
                     <ul>
                       <li v-for="activity in findActivities(category)" :key="activity.id">
                         {{ activity.title }}
                         <ul>
-                          <li v-for="scheduleEntry in activity.scheduleEntries().items" :key="scheduleEntry.id">
+                          <li v-for="scheduleEntry in activity.scheduleEntries().items"
+                              :key="scheduleEntry.id"
+                              :set="scheduleEntryItem = defineHelpers(scheduleEntry)">
```

Comment:
Diese Zeile verstehe ich nicht. Ist `:set` ein Konstrukt von Vue? Ist das ein Trick der `scheduleEntryItem` immer neu berechnet wenn die Ansicht neu gerendert wird / `scheduleEntry` sich ändert?
Falls letzteres, würde ich entweder das `<li>` in eine separate Komponente auslagern und `scheduleEntryItem` dort in einem `computed` berechnen, oder gleich hier in dieser Komponente ein `computed` einfügen:
```
scheduleEntryItems () {
  return this.activity.scheduleEntries().items.map(item => defineHelpers(item))
}
```
und dann darüber mit `v-for` iterieren